include/Core/Terms/Modules/DeleteTermsByPostCountModule.php

```diff
@@ -35,17 +35,21 @@ public function render() {
 
 			<?php _e( 'Delete Terms if the post count is ', 'bulk-delete' ); ?>
 			<?php $this->render_number_comparison_operators(); ?>
-			<input type="number" name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_post_count" placeholder="<?php _e( 'Post count', 'bulk-delete' ); ?>">
+			<input type="number" name="smbd_<?php echo esc_attr( $this->field_slug ); ?>" placeholder="<?php $markup = _e( 'Post count', 'bulk-delete' ); ?>" min="0">
```

Comment:
Are you sure the assignment to ``$markup`` is correct as ``_e('Post count', 'bulk-delete')`` seems to always return ``null``.